tests/containers/podman_integration.pm

```diff
@@ -105,19 +106,21 @@ sub run {
     # user / remote
     #
 
-    assert_script_run "cp -r test/system.orig test/system";
-    @skip_tests = split(/\s+/, get_required_var('PODMAN_BATS_SKIP') . " " . get_var('PODMAN_BATS_SKIP_USER_REMOTE', ''));
-    foreach my $test (@skip_tests) {
-        script_run "rm test/system/$test.bats";
-    }
+    unless (is_sle_micro('<5.5')) {
+        assert_script_run "cp -r test/system.orig test/system";
```

Comment:
This block is being used twice, almost identical. Perhaps we should refactor a subroutine?

Reply:
> This block is being used twice, almost identical. Perhaps we should refactor a subroutine?

One block runs as user and another as root.  Refactoring will bring little gain as we would have switch user/root terminal at least once more.

Reply:
We can create a subroutine that takes `@skip_tests` and `$log_file` as parameters.

I personally do think that a duplication of more than 10 LOC suggests a new subroutine though, however I leave it to your judgement, as this is a style question.

Reply:
> We can create a subroutine that takes `@skip_tests` and `$log_file` as parameters.
> 
> I personally do think that a duplication of more than 10 LOC suggests a new subroutine though, however I leave it to your judgement, as this is a style question.

I will try this if a new PR is needed for these tests on SLES.